finda-gateway/src/main/kotlin/finda/findagateway/global/filter/PassportGatewayFilterFactory.kt

```
@@ -0,0 +1,113 @@
+            if (isPermitAllPath) {
+                return@GatewayFilter chain.filter(exchange)
+            }
+
+            Mono.fromCallable {
+                val token = resolveToken(exchange)
+                val jwtClaims = jwtProvider.validateAccessToken(token)
+
+                // Authority 변환
+                val authority = try {
+                    Authority.valueOf(jwtClaims.userType)
+                } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)